assets/florence/pagedown/Markdown.Editor.js

```diff
@@ -1113,10 +1114,12 @@
                 text = text.replace(/^http:\/\/(https?|ftp):\/\//, '$1://');
                 if (!/^(?:https?|ftp):\/\//.test(text))
                     text = 'http://' + text;
+                if (check.checked == true)
+                    text = text.replace(/.*\/\/[^\/]*/, '')
```

Comment:
This regex seems greedy and should it be anchored?  - anchoring from 'start (`^`) will also protect against any URLs like: `https://foo.com//bar` (where the `//bar` onwards will be considered a new match and thus all deleted).

What I read this to mean: As many of any characters followed by `//` followed by as many characters except `/`
It is good practice (although not done everywhere to comment what the goal of the regex is above the regex line.

Reply:
So I will throw my hands in the air and admit I am no Regex Ninja so I pretty much googled "Regex that removes domain from url".

Happy to add the comment but unsure if you want me to change the Regex lol.

Reply:
No worries, I think but maybe worth testing, it would be slightly better to change to:
`/^.*\/\/[^\/]*/, ''` - just the 'start of line anchor `^` at the beginning.

Reply:
Done